.github/workflows/pull_request_tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.9"  # matches current Python in production
+          python-version: "3.11"  # matches current Python in production
```

Comment:
**thought:** For Nucleus we went straight to 3.12. I wonder if that's OK here, too

Reply:
Yeah I thought about that. But 3.12 is only on the 2nd patch release. I was thinking we would want bedrock to be a little more careful than the others.